AddQtAndroidApk.cmake

```diff
@@ -44,8 +45,12 @@ if(NOT QT_ANDROID_NDK_ROOT)
     endif()
 endif()
 string(REPLACE "\\" "/" QT_ANDROID_NDK_ROOT ${QT_ANDROID_NDK_ROOT}) # androiddeployqt doesn't like backslashes in paths
+set(QT_ANDROID_NDK_ROOT ${QT_ANDROID_NDK_ROOT} CACHE STRING "" FORCE)
 message(STATUS "Found Android NDK: ${QT_ANDROID_NDK_ROOT}")
 
+set(QT_ANDROID_CMAKE_FOUND ON CACHE BOOL "QtAndroidCMake have been found" FORCE)
```

Comment:
What's the reason for adding this variable? And why in cache (i.e. I can switch it OFF)?

Reply:
This variable doesn't have any reason to exist anymore. I used it before discovering usage of FetchContent in CMake.
This variable was helping me to mimic the behavior of FetchContent. In my CMake script i could check if QT_ANDROID_CMAKE_VERSION wasn't defined. If not then download the repository from github. I need to remove this variable.
You can see an example of my previous usage here: https://github.com/OlivierLDff/QaterialGallery/blob/d2f3187c0a15a421b435a7ad15efd75fa9687c5c/platform/PreBuildPlatform.cmake
Yes it's very ugly.